Geometry/HcalCommonData/src/HcalDDDRecConstants.cc

```diff
@@ -287,6 +287,32 @@ void HcalDDDRecConstants::getLayerDepth(const int ieta,
 #endif
 }
 
+int HcalDDDRecConstants::getLayerFront(const int idet, const int ieta,
+				       const int iphi, const int depth) const {
+  int subdet = (idet == 1) ? 1 : 2;
+  int eta    = (ieta > 0) ? ieta : -ieta;
+  int zside  = (ieta > 0) ? 1 : -1;
```

Comment:
It seems like you could define `zside` first, and then just define `int eta = ieta*zside;`, to avoid repeating the ternary operator.